charts/mla/minio-lifecycle-mgr/values.yaml

```diff
@@ -14,7 +14,7 @@
 lifecycleMgr:
   image:
     repository: docker.io/minio/mc
-    tag: latest
```

Comment:
can we use the version from the `charts/mla/minio` instead? That would be `RELEASE.2023-04-28T18-11-17Z` at this time

Reply:
That release does not seem to exist for `minio/mc`:
```
manifest for minio/mc:RELEASE.2023-04-28T18-11-17Z not found: manifest unknown: manifest unknown
```

Reply:
Personally I would not give much value on keeping mc on the same version as Minio. I would put more emphasis on adding a comment explaining why this value is overridden and that it is not _this_ *one* magic specific version, but just "latest MC at the time of making the PR" or something.

Reply:
that makes sense to me. Agree with @xrstf - please put a comment explaining and it's fine by me.

Reply:
Where would I put this comment? Into the release notes?

Reply:
In the values.yaml, right above the line where we specify the mc image.

Reply:
👍